var/spack/repos/builtin/packages/py-mne/package.py

```diff
@@ -23,8 +24,6 @@ class PyMne(PythonPackage):
     variant("hdf5", default=False, when="@1:", description="Enable hdf5 functionality.")
 
     depends_on("python@3.7:", when="@0.24:", type=("build", "run"))
```

Comment:
When you're removing these, you can also remove 3.7. Since Spack only supports 3.7+, it doesn't matter which version dropped support for 3.6.